profiles/resources/default.cfg

```diff
@@ -189,6 +189,7 @@ Wapiti_Unauth_____Wapiti_____@@@FRAMEWORK_DIR@@@/scripts/run_wapiti.sh @@@TOOL_W
 DirBusterNotInteractive_____DirBuster Not Interactive_____java -jar @@@TOOL_DIRBUSTER_BIN@@@ -u @@@target_url@@@ -H -t 20 -R -r '###plugin_output_dir###' -l @@@DICT_GENERIC_DIRBUSTER@@@ | grep -v "^java." | tr "\t" " " | grep -v "^ at" # Remove java exception garbage at the end
 DirBusterInteractive_____DirBuster_____java -jar @@@TOOL_DIRBUSTER_BIN@@@ -u @@@target_url@@@ -t 20 -R -r '###plugin_output_dir###' -l @@@DICT_GENERIC_DIRBUSTER@@@ | grep -v "^java." | tr "\t" " " | grep -v "^ at" # Remove java exception garbage at the end
 DirBuster_Extract_URLs_____Extract URLs_____@@@FRAMEWORK_DIR@@@/scripts/extract_urls.sh dirbuster DirBuster.txt @@@target_url@@@
+SVN_Extract_URLs_____Extract URLs_____cd @@@TOOL_SVN_EXTRACTOR_DIR@@@ ; python svn_extractor.py --url @@@target_url@@@  2>&1 | tee ###plugin_output_dir###/SVN_Report.txt
```

Comment:
@delta24 [here](http://pastebin.com/KDUudXXk) is the final code structure. The problem here is extracted urls of svn report is replaced by extracted urls of dirbutser report. What do you suggest ?